src/Kmd.Logic.Gateway.Automation/Gateway/Product.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kmd.Logic.Gateway.Automation.Gateway
+{
+    public class Product
+    {
+        public string Name { get; set; }
+
+        public string Logo { get; set; }
+
+        public string Documentation { get; set; }
+
+        public string PoliciesXmlFile { get; set; }
+
+        public string Description { get; set; }
+
+        public string Published { get; set; }
+
+        public bool? ApiKeyRequired { get; set; }
+
+        public bool? ClientCredentialRequired { get; set; }
+
+        public bool? ProviderApprovalRequired { get; set; }
+
+        public string LegalTerms { get; set; }
+
+        public string Visibility { get; set; }
+
+        public string OpenidConfigIssuer { get; set; }
+
+#pragma warning disable CA1056 // URI-like properties should not be strings
```

Comment:
So change it to Uri

Reply:
Product creation Input parameter is string not Uri type. while passing parameter again we have to convert it to string, So this has been considered as string

Reply:
I understand, but when you give here Uri type, then it's kind of a validation. When somebody tries to put other value than Uri, then it will fail at the beginning. But it is a suggestion. I am not going to push you ;)